tests/test_text_data.py

```diff
@@ -1,9 +1,24 @@
+import pytest
 from fastai import *
 from fastai.text import *
 
+@pytest.fixture(scope="module")
+def text_df(multilabel=False):
```

Comment:
could make this more generic (multilabel being the number of labels) - but have kept it clean and simpler, as focus is on testing a mutlilabel datatest and not really making it super generic

can update if you feel otherwise

Reply:
It's getting a bit long, so if it's not too much to do, I'd prefer one generic test.

Reply:
Ok 👍 
(I assume you meant this for combining singlelabel and multilabel tests into one - the other comment)

updating